apps/vs-code-react/src/app/designer/DesignerCommandBar/index.tsx

```diff
@@ -220,6 +222,26 @@ export const DesignerCommandBar: React.FC<DesignerCommandBarProps> = ({
         saveWorkflowMutate();
       },
     },
+    {
+      key: 'SaveBlank',
+      disabled: isDisabled,
+      text: Resources.UNIT_TEST_SAVE_BLANK,
+      ariaLabel: Resources.UNIT_TEST_SAVE_BLANK,
+      onRenderIcon: () => {
```

Comment:
is this working/looking fine in the command bar? As we are using new fluent ui for the command bar the `onRenderIcon` is not needed anymore. 
You can use the same thing as the normal save above
```
icon: isSavingBlankUnitTest ? <Spinner size="extra-small" /> : <SaveRegular />,
renderTextIcon: null,
```